certification/internal/engine/engine_test.go

```diff
@@ -81,6 +99,8 @@ var _ = Describe("Execute Checks tests", func() {
 				goodCheck,
 				errorCheck,
 				failedCheck,
+				optionalCheckPassing,
+				optionalCheckFailing,
```

Comment:
Is it worth it to add an 'InformationalCheck' result for things that will always pass but we want the user to be informed ex: the SCC check where we can show a help text if a custom scc is in use ?

Reply:
Let's discuss on that issue.